rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/NodeLabel.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive_grid_cloud_portal.rm.client.monitoring;
+
+import java.util.Map;
+
+import org.ow2.proactive_grid_cloud_portal.rm.client.NodeSource;
+import org.ow2.proactive_grid_cloud_portal.rm.client.NodeSource.Host.Node;
+
+import com.smartgwt.client.widgets.Label;
+
+
+/**
+ * @author ActiveEon Team
+ * @since 19/01/17
+ */
+public final class NodeLabel {
+
+    private NodeLabel() {
+    }
+
+    public static void update(Map<String, NodeSource> nodes, Label nodeLabelToUpdate, Node nodeToUpdate) {
```

Comment:
Would not it be better if the update method was in Node class instead of a service class ?